docs/quickstart.rst

```diff
@@ -144,3 +144,140 @@ Otherwise, models are represented as dicts.
 
     Values in ``request.swagger_data`` are only available if
     ``pyramid_swawgger.enable_request_validation`` is enabled.
+
+Accessing Swagger Operation
+---------------------------
+
+During the implementation of an endpoint you could eventually have need of accessing the Swagger Specs that defined that specific view.
+``pyramid_swagger`` will inject in the request object a new property (that will be evaluated only if accessed) called ``operation``.
+
+``request.operation`` will be set to ``None`` for Swagger 1.2 defined endpoints, while it will be an `Operation <https://github.com/Yelp/bravado-core/blob/v4.8.4/bravado_core/operation.py#L12>`_ object if the endpoint is defined by Swagger 2.0 specs.
+
+Set pyramid view renderer
+-------------------------
+
+Using :mod:`pyramid_swagger` you will get automatic conversions between input JSON objects to easy to handle python objects.
+An example could be a swagger object property marked to use date property, the library will take care of converting the ISO 8601 date representation to an easy to handle python ``datetime.date`` object.
```

Comment:
Hrm. Is there any way for us to avoid having to re-write every renderer to introduce the "right" pyramid-swagger behavior here? Ideally we'd want to do some sort of composition here where you combine a default pyramid renderer with pyramid-swagger marshalling. 

What if we write a single new swagger_view_config decorator that accepts a pass-through renderer option and the only thing it does is unmarshall any objects it has to_wire methods for? Then you would just rewrite all existing view_config decorators to be swagger_view_config decorators and nothing else would change?

Reply:
Pyramid docs for reference https://docs.pylonsproject.org/projects/pyramid/en/latest/narr/renderers.html#renderers-chapter

Reply:
I was thinking to different approaches to achieve this result:
 - intercepting ``BeforeRender`` pyramid event ([doc](https://docs.pylonsproject.org/projects/pyramid-cookbook/en/latest/templates/templates.html))
 - "overriding" ``view_config`` decorator
 - modifying the ``Request``/``Response`` object propagated to views
 - defining a new Renderer (chosen approach)

The main reason why I discarded the other approaches was because this seemed to be the less invasive and back compatible one.
Here more detailed reasons:
 - ``BeforeRenderer`` event contains the value to render, but the passed variable could be any type of object returned by the view. This means that if we perform unmarshaling with the event listener then the object that will be rendered will be "modified" only in case of a dicitonary. Since the approach was not generalizable to all the possible valid responses I decided to move on. Additionally event listeners are not supposed to modify data.
 - overriding ``view_config`` will require developers to change the decoration system used in their services. Additionally ``view_config`` decorator is a delicate piece of pyramid framework and I guess that it could be changed/improved in the feature, so it could represent an overhead for the maintenance of pyramid_swagger
 - modifying the ``Request``/``Response`` seemed to be a valid solution, but the code was starting to be too complicated, so I decided to stop 😄 
 - defining a new renderer allows developers to easily plug the renderer if they want to.
    Marshaling should have a really small performance impact, but I would prefer to avoid to force all the users to use it.
    Additionally applying this new renderer will be a few line changes for the majority of the systems that I've worked on.
    In general an API service using pyramid_swagger defines a default renderer to be used if no renderer is specified in ``view_config`` decorator.